modules/dnn/src/dnn.cpp

```diff
@@ -2980,6 +2980,18 @@ int Net::getLayerId(const String &layer)
     return impl->getLayerId(layer);
 }
 
+String parseLayerParams(const String& name, const LayerParams& lp) {
+    DictValue param = lp.get(name);
+    std::ostringstream out;
+    out << name << " ";
+    switch (param.size()) {
+        case 1: out << ": " << param << "\\l"; break;
+        case 2: out << "(HxW): " << param.get<int>(0) << " x " << param.get<int>(1) << "\\l"; break;
+        case 3: out << "(DxHxW): " << param.get<int>(0) << " x " << param.get<int>(1) << " x " << param.get<int>(2) << "\\l"; break;
+    }
```

Comment:
`default:` ?

Lets don't try to process silently of unsupported cases.

Reply:
Maybe we need to use switch only for prefix but all the values print in the loop over `param.size()`?